docs/src/main/paradox/release-notes/10.1.x.md

```diff
@@ -2,11 +2,37 @@
 
 ## 10.1.9
 
+10.1.9 is the tenth release in the 10.1.x series of Akka HTTP. See the [announcement](https://akka.io/blog/news/2019/07/17/akka-http-10.1.9-released.html) and the full list of changes below.
+
 ### Changes since 10.1.8
 
+For a full overview you can also see the [10.1.9 milestone](https://github.com/akka/akka-http/milestone/10.1.9?closed=1):
+
+##### akka-http-core
```

Comment:
Should this and the relevant below be `####` vs `#####`?

Reply:
I think we sometimes split things up between 'improvements' and 'bugfixes', and always use `#####` whether or not we have that split for that release.